src/platforms/dialog-flow/directives.ts

```diff
@@ -194,3 +196,105 @@ export class MediaResponse implements IDirective {
     (reply as DialogFlowReply).payload.google.richResponse = richResponse.add(mediaResponse);
   }
 }
+
+export class PermissionsDirective implements IDirective {
+  public static platform: string = "dialogFlow";
+  public static key: string = "dialogFlowPermission";
+
+  public constructor(public permissionOptions: PermissionOptions) { }
+
+  public async writeToReply(reply: IVoxaReply, event: IVoxaEvent, transition: ITransition): Promise<void> {
+    (reply as DialogFlowReply).fulfillmentText = "login";
+    const permission: Permission = new Permission(this.permissionOptions);
+
+    const google: any = (reply as DialogFlowReply).payload.google;
+    google.systemIntent = {
+      data: permission.inputValueData,
+      intent: permission.intent,
+    };
+  }
+}
+
+export class DateTimeDirective implements IDirective {
+  public static  platform: string = "dialogFlow";
+  public static key: string = "dialogFlowCard";
+
+  constructor(public dateTimeOptions: DateTimeOptions) {}
+
+  public async writeToReply(reply: IVoxaReply, event: IVoxaEvent, transition: ITransition): Promise<void> {
+    const google: any = (reply as DialogFlowReply).payload.google;
+    const dateTime = new DateTime(this.dateTimeOptions);
+    google.expectedInputs = {
+      inputValueData: dateTime.inputValueData,
+      intent: dateTime.intent,
+    };
+
+  }
+}
+
+export class ConfirmationDirective implements IDirective {
+  public static  platform: string = "dialogFlow";
```

Comment:
There's an extra space before `platform`, here and in other files as well

Reply:
@omenocal fixed :P